pkg/operator/configobservation/scheduler/observe_scheduler.go

```diff
@@ -39,28 +42,67 @@ func ObserveSchedulerConfig(genericListers configobserver.Listers, recorder even
 		errs = append(errs, err)
 		return prevObservedConfig, errs
 	}
-	configMapName := schedulerConfig.Spec.Policy.Name
 
-	switch {
-	case len(configMapName) == 0:
-		sourceTargetLocation = resourcesynccontroller.ResourceLocation{}
-	case len(configMapName) > 0:
-		sourceTargetLocation = resourcesynccontroller.ResourceLocation{
-			Namespace: operatorclient.GlobalUserSpecifiedConfigNamespace,
-			Name:      configMapName,
+	// Deprecated handling for Policy config
+	// TODO(@damemi): Remove this when Policy API is removed
+	if len(schedulerConfig.Spec.Policy.Name) > 0 {
```

Comment:
Maybe we should warn in the logs at least about this fact? Also, make sure to create an entry in the release notes for 4.7 mentioning the fact that we're switching from policy to profiles.

Reply:
Both good points, how do we make an entry in the release notes?